.github/workflows/docker.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Docker Workflow
+on:
+  push:
+    branches: [master]
```

Comment:
do we want to rebuild the container for every document push to master?

Reply:
That's precisely why the results are cached in the workflow and why I called it out explicitly in the PR and commit.